Sources/ClientRuntime/Networking/Http/StreamableHttpBody.swift

```diff
@@ -51,7 +51,10 @@ class StreamableHttpBody: IStreamable {
     ///   - streamSeekType: type of seek
     func seek(offset: Int64, streamSeekType: AwsCommonRuntimeKit.StreamSeekType) throws {
         guard streamSeekType == .begin else {
-            throw StreamError.notSupported("Seeking from end is not supported. Only seeking from beginning is supported.")
+            throw StreamError.notSupported("""
+            Seeking from end is not supported.
+            Only seeking from beginning is supported.
+            """)
```

Comment:
I find it a bit odd to have an error message with newlines like this. Imo the original is preferable.

Reply:
rather wanted to concat. didn't realize until saw you comment. Updated.